common/src/main/java/com/redhat/cloud/notifications/routers/dailydigest/TriggerDailyDigestRequestDto.java

```diff
@@ -0,0 +1,88 @@
+package com.redhat.cloud.notifications.routers.dailydigest;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+
+import javax.validation.constraints.NotBlank;
+import java.beans.ConstructorProperties;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.ZoneOffset;
+import java.util.Objects;
+
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+public class TriggerDailyDigestRequestDto {
+
+    @JsonProperty("application_name")
```

Comment:
All right, removed the redundant annotations!

Reply:
I am going to unresolve this one, since now the tests are telling me the following thing:

```
com.fasterxml.jackson.databind.exc.InvalidDefinitionException: Cannot construct instance of `com.redhat.cloud.notifications.models.AggregationCommand` (no Creators, like default constructor, exist): cannot deserialize from Object value (no delegate- or property-based Creator)
 at [Source: (String)"{"aggregationKey":{"orgId":"org-id","bundle":"bundle-name","application":"application-name"},"start":"2023-03-08T10:43:33.55235369","end":"2023-03-13T10:43:33.55235369","subscriptionType":"DAILY"}"; line: 1, column: 2]
```

So I am guessing that that might be why I annotated everything...

Reply:
Ok then feel free to put back the annotation.